server/user.go

```diff
@@ -528,18 +528,22 @@ func replyDelUser(s *Session, msg *ClientComMessage) {
 	}
 }
 
-type userUpdate struct {
-	// Single user ID being updated
-	uid types.Uid
-	// User IDs being updated
-	uidList []types.Uid
+// UserCacheReq contains data which mutates one or more user cache entries.
+type UserCacheReq struct {
+	// Name of the node sending this request in case of cluster. Not set otherwise.
+	Node string
+
+	// UserId is set when a single user is updated.
+	UserId types.Uid
```

Comment:
Why not use UserIdList[0] in such a case?

Reply:
It's two different requests. `UserIdList` is used to increment or decrement count of topics a user is subscribed to. `UserId` is used to increment or decrement the count of unread messages for a give user. It's possible to use `UserIdList[0]` instead of `UserId` but then I would still have to use some heuristics to separate these use cases.

Reply:
Makes sense. Can you update the comments to make it explicit please?